### PR TITLE
add Release drafter

### PR DESCRIPTION
**Pull request by casz**
Here is our checklist for contributors. No hard requirement here, just a reminder

- [x] Please describe what you did

Since we already use PR squashing in most cases we can easily introduce [release-drafter](https://github.com/toolmantim/release-drafter) to manage our changelog.

~Do not merge before everything is moved over to releases tabs~ :smile: 

~I will create a separate PR with all the changelog moved to GitHub releases.~

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2-feature-request.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: "🚀 Feature request"
+about: I have a suggestion
+---
+
+### Your checklist for this issue
+
+🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.
+
```
Comment (casz) [inline .github/ISSUE_TEMPLATE/2-feature-request.md]: Suggestions? :smile: or just remove the feature request one?

File: .github/ISSUE_TEMPLATE/3-plugin-compatiblity.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+name: "💥 Plugin compatibility"
+about: Create a plugin compatibility issue report
+---
+
+### Your checklist for this issue
+
+🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.
+
+- [ ] Create an issue on [issues.jenkins-ci.org](https://issues.jenkins-ci.org), that is commonplace to report plugin issues
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/3-plugin-compatiblity.md]: This doesn't really make sense to me:
> that is commonplace to report plugin issues

Maybe something like this?
```suggestion
- [ ] Create an issue on [issues.jenkins-ci.org](https://issues.jenkins-ci.org), set the component to the plugin you are reporting it against
```

Comment (casz) [reply .github/ISSUE_TEMPLATE/3-plugin-compatiblity.md]: :+1:

**Comment by ewelinawilkosz** [comment]
great idea, but since I've never used it before I have a question - how will that affect old changelog? We haven't had nice rules/requirements before, so will it make all the old releases changelog messy?

**Comment by casz** [comment]
See new changelog here: https://github.com/jenkinsci/configuration-as-code-plugin/releases

As maintainers we can edit PR titles and add labels as we see fit. Once drafter sees a new push to GitHub master it redraft all PRs merged to master from my understanding of the code inside the project.

**Comment by casz** [comment]
Moved the docs to: #803

**Pull request merged**